src/Microsoft.Docs.LearnValidation/DependencyMap/DependencyItem.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TripleCrownValidation.DependencyMap
+{
+    public class DependencyItem
```

Comment:
[JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]

or we do not want to touch the code? Just import them